src/EnergyPlus/BoilerSteam.cc

```diff
@@ -208,11 +208,11 @@ namespace BoilerSteam {
             {
                 auto const SELECT_CASE_var(DataIPShortCuts::cAlphaArgs(2));
 
-                if ((SELECT_CASE_var == "ELECTRICITY") || (SELECT_CASE_var == "ELECTRIC") || (SELECT_CASE_var == "ELEC")) {
+                if (SELECT_CASE_var == "ELECTRICITY") {
```

Comment:
Remove input synonyms here and many other places. Don't move to standard function yet - save that for a later round of changes. And also keep some "old" output resource types.